bench/Chainweb/Pact/Backend/ForkingBench.hs

```diff
@@ -330,11 +327,10 @@ withResources trunkLength logLevel f = C.envWithCleanup create destroy unwrap
         rocksDbAndDir <- createRocksResource
         payloadDb <- createPayloadDb
         blockHeaderDb <- testBlockHeaderDb (snd rocksDbAndDir) genesisBlock
-        tempDir <- fst <$> newTempDir
         coinAccounts <- newMVar mempty
         nonceCounter <- newIORef 1
         txPerBlock <- newIORef 10
-        sqlEnv <- startSqliteDb cid logger tempDir False
+        sqlEnv <- openSQLiteConnection "" {- temporary SQLite db -} chainwebPragmas
```

Comment:
This is the main change in this file: Passing "" as filename to `openSQLiteConnection` let's us remove the `tempDir` from the `Resources` environment. 

This also fixes a bug: the code didn't delete the `tempDir` at the end of the benchmark execution, thus leaking the temporary sqlite databases.